src/test/javascript/spec/component/complaints/complaint-response.service.spec.ts

```diff
@@ -12,29 +12,31 @@ import { User } from 'app/core/user/user.model';
 import { TextExercise } from 'app/entities/text-exercise.model';
 
 describe('ComplaintResponseService', () => {
-    let testBed: TestBed;
     let complaintResponseService: ComplaintResponseService;
     let httpTestingController: HttpTestingController;
     let defaultComplaintResponse: ComplaintResponse;
+    let accountService: AccountService;
     let expectedComplaintResponse: any;
 
     beforeEach(() => {
-        TestBed.configureTestingModule({
+        return TestBed.configureTestingModule({
```

Comment:
```suggestion
        TestBed.configureTestingModule({
```
The `return` should’t be needed here, I think.